mayastor/src/bdev/nexus/nexus_channel.rs

```diff
@@ -144,7 +146,9 @@ impl NexusChannel {
             .filter(|c| c.status() == ChildStatus::Online)
             .map(|c| {
                 channels.ch.push(
-                    BdevHandle::try_from(c.get_descriptor().unwrap()).unwrap(),
+                    BdevHandle::try_from(c.get_descriptor().unwrap())
+                        .unwrap()
+                        .set_retries(max_io_attempts),
```

Comment:
Im not quite sure why this is added here....

Reply:
Jonathan's change was to have the max number of retries stored in the BdevHandle. I wanted to be consistent with that, rather than having one place for nexus IO and another for internal IO.

Reply:
Changed to use just the value stored in the nexus.